src/repl.ts

```diff
@@ -266,7 +266,7 @@ function executeFile() {
     if (!editor) {
         return;
     }
-
+    
```

Comment:
Not important, but there also doesn't need to be an edit here, right?

Reply:
I suggest do a pr removing all the training spaces in source files.
Reasons are:
1. Some plugins highlight trailing spaces. Keeping them is not good looking.
2. Some configurations can remove trailing spaces when saving. It will be a pain to retain thoes spaces.